app/views/option_sets/_options.html.erb

```diff
@@ -15,7 +15,7 @@
 
 <% unless read_only || @option_set.huge? %>
   <%# edit option modal %>
-  <div class="modal fade edit-named-item" id="edit-option" tabindex="-1" role="dialog" aria-labelledby="editOption" aria-hidden="true">
+  <div class="modal fade new-modal-style edit-named-item" id="edit-option" tabindex="-1" role="dialog" aria-labelledby="editOption" aria-hidden="true">
```

Comment:
I think I see what is going on here now. But I'm not sure you need this. Would it hurt if you just used .modal instead of .new-modal-style? If you search the project for `class="modal` you will see all the places where bootstrap modals are used. I _think_ that they are all pretty much the same. That should certainly be the goal. We can discuss more if you like.